source/profiler/index.js

```diff
@@ -38,16 +38,26 @@ exports.handler = async (event) => {
       event[key] = data.Item[key];
     });
     let mediaInfo = JSON.parse(event.srcMediainfo);
-    event.srcHeight = mediaInfo.video[0].height;
-    event.srcWidth = mediaInfo.video[0].width;
+    event.rotation = mediaInfo.video[0].rotation;
```

Comment:
Hi @arivera-skycred,
Can you confirm this pull request is up-to-date?
This change you introduced requires the rotation property returned by mediainfo to be included in the event, but that file is not included in this PR.